file-formats/sajc/type/zif_aff_sajc_v1.intf.abap

```diff
@@ -64,9 +61,38 @@ INTERFACE zif_aff_sajc_v1
   "! <p class="shorttext">Section Name</p>
   "! Name of the section
   TYPES ty_section_name TYPE c LENGTH 10.
+  "! <p class="shorttext">Parameter Title Text</p>
+  "! Text of the parameter title
+  TYPES ty_parameter_title_text TYPE c LENGTH 255.
   "! <p class="shorttext">Title Text</p>
   "! Text of the title
   TYPES ty_title_text TYPE c LENGTH 120.
+  "! <p class="shorttext">Radio Button Group</p>
+  "! Name of the radio button group
+  TYPES ty_radiobutton_group TYPE c LENGTH 4.
+  "! <p class="shorttext">Screen Element</p>
+  "! Type of the screen element
+  "! $values {@link zif_aff_sajc_v1.data:co_screen_element}
+  "! $default {@link zif_aff_sajc_v1.data:co_screen_element.none}
+  TYPES ty_screen_element TYPE c LENGTH 1.
+
+  CONSTANTS:
+    "! <p class="shorttext">Screen Element</p>
+    "! Type of the screen element
+    BEGIN OF co_screen_element,
+      "! <p class="shorttext">None</p>
+      "! None
+      none        TYPE ty_screen_element VALUE ' ',
+      "! <p class="shorttext">Checkbox</p>
+      "! Checkbox
+      checkbox    TYPE ty_screen_element VALUE 'C',
+      "! <p class="shorttext">Radio Button</p>
+      "! Radio button
+      radiobutton TYPE ty_screen_element VALUE 'R',
```

Comment:
Minor: Does it make sense to change it to `radio_button`?

```suggestion
      radio_button TYPE ty_screen_element VALUE 'R',
```

Reply:
Yes, these changes make sense. I will change it.

Reply:
I changed it.